.vscode/launch.json

```diff
@@ -3,8 +3,9 @@
   "configurations": [
     {
       "name": "Debug Extension",
-      "type": "extensionHost",
+      "type": "pwa-extensionHost",
```

Comment:
My google skills are failing me—what's the difference between `extensionHost` and `pwa-extensionHost` here?

Reply:
I'm curious too, but hadn't even tried to Google it yet. 🙂

Reply:
It targets a newer javascript debugger that has a richer feature set -- the regular `extensionHost` type doesn't support the `autoAttachChildProcesses` configuration. The newer debugger is installed by default in recent versions of vscode -- people on really old versions of vscode would need to install it manually

Honestly, the documentation seems to be pretty sparse. Here's the newer debugger library: https://github.com/microsoft/vscode-js-debug

Let me know if you have more questions -- I can dig into the specifics if there are concerns.

Reply:
Nope, that makes sense—thanks for the explainer! 😄